samples/quick-start/Program.cs

```diff
@@ -23,7 +23,7 @@ class Program {
 			
 			#region createClient
 			var settings = EventStoreClientSettings
-				.Create("{connectionString}");
+				.Create("esdb://admin:changeit@localhost:2113?tls=false");
```

Comment:
**FIX:** We might need to keep it like that, as without that the connection string replacement won't work in docs. Right, @alexeyzimarev?

Reply:
As long as everything stays in the same region we could just move all of this outside `Main`

Reply:
Yup, it's not ideal, but I think that we can keep this snippet just to have a compile-time check.